src/Sylius/Bundle/ApiBundle/composer.json

```diff
@@ -54,6 +54,9 @@
     "extra": {
         "branch-alias": {
             "dev-main": "2.0-dev"
+        },
+        "symfony": {
+            "require": "7.0.*"
```

Comment:
```suggestion
            "require": "^7.0"
```

In every bundle

Reply:
I would be even for bumping the minimum version to ^7.1, but definitely in a separate PR after merge this one 🖖🏻